admin/.nvmrc

```diff
@@ -0,0 +1 @@
+v14.17.0
```

Comment:
Does it make sense to upgrade the version as soon as possible or why using such an old version?